packages/angular-material/README.md

```diff
@@ -1,32 +1,27 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
 
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
 
-# Angular Material Renderers Package
+## Angular Material Renderers Package
 
-This is the JSONForms Angular Material renderers package. This package only contains renderers and must be combined with [JSONForms Angular](https://github.com/eclipsesource/jsonforms/blob/master/packages/angular).
+This is the JSONForms Angular Material renderers package. This package only contains renderers and must be combined with [JSON Forms Angular](https://www.npmjs.com/package/@jsonforms/angular).
```

Comment:
The previous link was fine ;) We don't need links to the npm packages. Just make sure that all links in all Readme and other Markdown files are absolute.